.github/ISSUE_TEMPLATE/introduce-sgid-dataset.md

```diff
@@ -0,0 +1,76 @@
+---
+name: Introduce data to the SGID
+about: Creation Checklist
+title: Add <dataset name> from <place>
+labels: 'introduction'
+assignees: ''
+---
+
+<!-- 
+Introduce your data, where it comes from, why it is being added etc
+-->
+# Summary
+
+We received yada yada from blahbidy blah. Foo Bar is the data contact and it will be updated quarterly. I propose we call it Baz and put it in this ISO category or group.
+
+<!-- 
+When the champion from your team has completed the triage, check [x] the checkbox
+-->
+
+# Triage
+
+- [ ] Data Team Triage Complete
+- [ ] Dev Team Triage Complete (assigned to)
+- [ ] Cadastre Team Triage Complete
+
+# We are introducing data
+
+The data was or will be added on `2020/00/00` to the following areas
+
+<!-- 
+add an [x] to the applicable areas you plan to add the data or ~strike~ out thoses that do not apply.
+If adding to the internal SGID with the hopes that swapper and forklift will push the item to SGID10 and AGOL
+you can strike them out. They are available if for some reason the internal sgid is skipped. 
+-->
+
+- [ ] Internal SGID
+- [ ] SGID10
+- [ ] ArcGIS Online
+
+<!--
+Where do we expect the data to show up. Check [x] all the areas 
+-->
+# The data should propagate to
+
+- [ ] ArcGIS Online
+- [ ] Internal SGID
+- [ ] SGID10
+- [ ] Open SGID
+- [ ] Open data
+- [ ] gis.utah.gov data pages
+- [ ] Stewardship doc and sgid-index
+- [ ] `SGID.META.AGOLItems`
+- [ ] `SGID.META.ChangeDetection`
+
+<!--
+assign yourself or someone to check that the dataset is live in its area. once verified, add the date of verification `2020/01/01`
+-->
+# We have verified the data is live 
+
+- [ ] ArcGIS Online (@assigned on ``)
+- [ ] Internal SGID (@assigned on ``)
+- [ ] SGID10 (@assigned on ``)
+- [ ] Open SGID (@assigned on ``)
+- [ ] Open data (@assigned on ``)
+- [ ] gis.utah.gov data pages (@assigned on ``)
+- [ ] Stewardship doc and sgid-index (@assigned on ``)
+- [ ] `SGID.META.AGOLItems` (@assigned on ``)
+- [ ] `SGID.META.ChangeDetection` (@assigned on ``)
+
+
+# Notification
+
+- [ ] Blog post (@assigned)
+- [ ] Twitter (@steveoh)
+  - [ ] Intent
```

Comment:
thought: `Intent` makes sense for data that will be deleted because it gives people a chance to prepare. However, I'm not sure I see a use case for `Intent` when adding data.